test/unit/product.js

```diff
@@ -300,6 +300,39 @@ describe('Product class', () => {
     });
   });
 
+  describe('get isButton', () => {
+    it('isButton getter is true when isButton is turn on and there is no button', () => {
```

Comment:
minor nitpick: don't need to repeat getter name here (should read `it('is true...')`)